aicsimageio/writers/timeseries_writer.py

```diff
@@ -0,0 +1,209 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import dask.array as da
+import numpy as np
+from fsspec.implementations.local import LocalFileSystem
+from imageio import get_writer
+
+from .. import types
+from ..dimensions import DimensionNames
+from ..exceptions import InvalidDimensionOrderingError, UnexpectedShapeError
+from ..readers import DefaultReader
+from ..transforms import reshape_data
+from ..utils import io_utils
+from .writer import Writer
+
+###############################################################################
+
+
+class TimeseriesWriter(Writer):
+    """
+    A writer for timeseries Greyscale, RGB, or RGBA image data.
+    Primarily directed at formats: "gif", "mp4", "mkv", etc.
+    """
+
+    _TIMEPOINT_DIMENSIONS = [
+        DimensionNames.Time,
+        DimensionNames.SpatialY,
+        DimensionNames.SpatialX,
+    ]
+    _TIMEPOINT_WITH_SAMPLES_DIMENSIONS = _TIMEPOINT_DIMENSIONS + [
+        DimensionNames.Samples
+    ]
+
+    DIM_ORDERS = {
+        3: "".join(_TIMEPOINT_DIMENSIONS),  # Greyscale
+        4: "".join(_TIMEPOINT_WITH_SAMPLES_DIMENSIONS),  # RGB / RGBA
+    }
+
+    @staticmethod
+    def _write_chunks(
+        f,
+        extension: str,
+        imageio_mode: str,
+        fps: int,
+        data: types.ArrayLike,
+        dim_order: str,
+    ):
+        with get_writer(
+            f,
+            format=extension,
+            mode=imageio_mode,
+            fps=fps,
+        ) as writer:
+            # Make each chunk of the dask array be a frame
```

Comment:
what do you mean by a "frame"?

Reply:
Frame of a gif. Frame of a movie. Etc. This is the timeseries writer so it basically means "each iter of the T index"